src/test/java/uk/nhs/digital/iucds/middleware/client/SftpClientTest.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.digital.iucds.middleware.client;
+
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Test;
+
+public class SftpClientTest {
+
+  private static final String HTML_FILE = "src/test/resources/input.html";
+  private static final String PRIVATE_KEY_PATH = "target/test/resources/id_rsa";
```

Comment:
ideally the key will be at ~/.ssh/ directory but for testing it is fine to have it bundled with source